packages/types/src/interfaces/eth/rpc.ts

```diff
@@ -178,6 +178,24 @@ export const rpc: DefinitionsRpc = objectSpread({}, netRpc, web3Rpc, {
     ],
     type: 'Bytes'
   },
+  feeHistory: {
+    description: "Returns fee history for given block count & reward percentiles",
+    params: [
+      {
+        name: "blockCount",
+        type: "U256"
+      },
+      {
+        name: "newestBlock",
+        type: "BlockNumber"
+      },
+      {
+        name: "rewardPercentiles",
+        type: "Option<Vec<f64>>"
```

Comment:
Need to look at this type - not sure we have an f64 inside the API. (I know the `f*` have been added recently as a hack-y thing to scale-codec)

Reply:
Relevant scale-codec PR - https://github.com/paritytech/parity-scale-codec/pull/340